### PR TITLE
veclibfort: migrate from homebrew/science

### PR DESCRIPTION
**Pull request by fxcoudert**
See https://github.com/Homebrew/homebrew-core/issues/15619

### CODE REVIEW

**Comment by fxcoudert** [comment]
`GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)`: yes, but definitely very useful for compiling Fortran scientific software, 15 science formulas depend on it (some of which will soon be migrated to core).

File: Formula/veclibfort.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  def install
+    system "make", "all"
+    system "make", "PREFIX=#{prefix}", "install"
+    pkgshare.install "tester.f90"
+  end
+
+  test do
+    ENV.fortran
+    system ENV.fc, "-o", "tester", "-O", pkgshare/"tester.f90", "-L#{lib}", "-lvecLibFort"
+    system "./tester"
```
Comment (ilovezfs) [inline Formula/veclibfort.rb]: Can we have this assert_match?

File: Formula/veclibfort.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class Veclibfort < Formula
+  desc "GNU Fortran compatibility for Apple's vecLib"
+  homepage "https://github.com/mcg1969/vecLibFort"
+  url "https://github.com/mcg1969/vecLibFort/archive/0.4.2.tar.gz"
+  sha256 "c61316632bffa1c76e3c7f92b11c9def4b6f41973ecf9e124d68de6ae37fbc85"
+  head "https://github.com/mcg1969/vecLibFort.git"
+
+  depends_on :fortran
+  depends_on "gcc"
```
Comment (ilovezfs) [inline Formula/veclibfort.rb]: This is redundant.

Comment (fxcoudert) [inline Formula/veclibfort.rb]: Can't `:fortran` be satisfied by setting `FC` to some other compiler?

File: Formula/veclibfort.rb (hunk 1 above)
Comment (fxcoudert) [inline Formula/veclibfort.rb]: It's gonna output numerical values which can vary… checking the return code as zero is what I want.

**Review by ilovezfs — APPROVED** [review]

**Pull request closed without merging**